modules/influxdb/values.yaml

```diff
@@ -0,0 +1,37 @@
+#architecture cannot be "high-availability" unless kubernetes persistent volume claim access mode is "ReadWriteMany"
+architecture: "standalone"
+database: "monitoring_data"
+adminUser:
+  name: "root"
+  pwd: "root"
+user:
+  name: "username"
+  pwd: "password"
+readUser:
+  name: "readuser"
+  pwd: "password"
+writeUser:
+  name: "writeuser"
+  pwd: "password"
```

Comment:
I think these should(?) be set as sensitive with `set_sensitive` inside the  helm_release resource block